### PR TITLE
doc: add AMO & Readme description

### PR DESCRIPTION
**Pull request by rugk**
Fixes https://github.com/rugk/unicodify/issues/20

### CODE REVIEW

**Comment by rugk** [comment]
TODO:
* [ ]  list all context menu options as text, too (this is likely good for SEO)

**Review by tdulcet — CHANGES_REQUESTED** [review]
Thanks for doing this! I think the add-on is finally ready to be published.

File: README.md [hunk 1]
```diff
@@ -8,13 +8,12 @@
 <img height="200" width="200" src="assets/header.svg">
 
 This is a (Firefox and Thunderbird) add-on (WebExtension) that allows you to autocorrect common text sequences and convert text characters to a look like a special font.
-For instance, it converts quotes like `"these"` to `“these”`, which are typographically more correct.
+For instance, it converts quotes like `"these"` to `“these”`, which are typographically correct.
 
-In contrast to many other add-ons, …
+Additionally, you can convert text into more than 20 different font styles and camel casing changes.
```
[the remark this answers is not in the document]
Comment (rugk) [reply README.md:13]: oh indeed, yeah I did not look at the add-on when writing this :sweat_smile:

**Comment by rugk** [comment]
> We should consider adding a donate button/link to the AMO listing

Hmm, I'd be reluctant to introduce money here, especially as we are more than one author and it's hard to distribute – unless you just link to both e.g. GitHub Sponsor profiles.

**Comment by tdulcet** [comment]
> Hmm, I'd be reluctant to introduce money here, especially as we are more than one author and it's hard to distribute – unless you just link to both e.g. GitHub Sponsor profiles.

I would be fine with a 50/50 split of any donations. GitHub Sponsors would also be great, especially since they have no fees and now support both one time and recurring donations. I already have a [GitHub Sponsors profile](https://github.com/sponsors/tdulcet), although it does not look like you do yet.

File: assets/texts/en/amoDescription.html [hunk 2]
```diff
@@ -1 +1,103 @@
...
+<ul>
+<li>📃 <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#translations">Translate this add-on into multiple languages!</a></li>
+<li>🐛 <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#coding">Fix some easy issues and get started in add-on development</a> (or just try out the development version)</li>
+<li>💡 <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#need-ideas">Or check out some other add-on issues</a> (or translate them).</li>
+</ul>
+Or, in any case, <a href="https://github.com/rugk/unicodify/blob/master/CONTRIBUTING.md#support-us">support us by spreading the word!</a> ❤️
+
+
+
+<b>🗯️ Permissions 🗯️</b>
```
Comment (tdulcet) [inline assets/texts/en/amoDescription.html:101]: May I suggest a different emoji here then the “[Anger Bubble](https://emojipedia.org/right-anger-bubble/)”: 🗨️, 💬, 📋 or ☑️.

File: assets/texts/en/amoDescription.html [hunk 3]
```diff
@@ -1 +1,103 @@
...
+
+
+<b>📝 Development 📝</b>
+The add-on is free/libre open-source software and developed on GitHub. <a href="https://github.com/rugk/unicodify">Fork it on GitHub</a> and contribute.
+<a href="https://github.com/rugk/unicodify/contribute">There are some easy issues to start with.</a>
+
+This extension only works with modern Firefox and Thunderbird v78 or higher.
+
+
+<b>🐬 Contribute 🐬</b>
```
Comment (tdulcet) [inline assets/texts/en/amoDescription.html:90]: Maybe use one of these emojis instead: 🙋, 👋, 📝, 👨‍💻 or 🆓, as I am not sure how the dolphin relates to contributing.

**Review by tdulcet — CHANGES_REQUESTED** [review]

File: assets/texts/en/amoDescription.html [hunk 4]
```diff
@@ -1 +1,103 @@
-export your Readme here, and potentially adjust it…
+This is a (Firefox and Thunderbird) add-on (WebExtension) that allows you to autocorrect common text sequences and convert text characters to a look like a special font.
+For instance, it converts quotes like <code>"these"</code> to <code>“these”</code>, which are typographically correct.
```
[the remark this answers is not in the document]
Comment (rugk) [reply assets/texts/en/amoDescription.html:2]: The translation is live on AMO, so it seems to work in https://addons.mozilla.org/firefox/addon/unicodify-text-transformer/

:shrug:

File: assets/texts/en/amoDescription.html (hunk 2 above)
Comment (rugk) [reply assets/texts/en/amoDescription.html:101]: oh lol, never noticed that, yeah probably… :sweat_smile: 

Also needs to be changed for all add-ons…

File: assets/texts/en/amoDescription.html (hunk 3 above)
Comment (rugk) [reply assets/texts/en/amoDescription.html:90]: well… dolphins are… dunno… I have no idea what I took when choosing these emojis :stuck_out_tongue_winking_eye:

**Review by tdulcet — DISMISSED** [review]
The add-on itself also needs a better description: 
image

I am not sure if that should be included in this PR.

**Review by tdulcet — APPROVED** [review]

**Pull request merged**